Index: pacemaker-1.0-Pacemaker-1.0.9.1/configure.ac
===================================================================
--- pacemaker-1.0-Pacemaker-1.0.9.1.orig/configure.ac
+++ pacemaker-1.0-Pacemaker-1.0.9.1/configure.ac
@@ -22,6 +22,7 @@ dnl     checks for system services
 AC_INIT(pacemaker, 1.0.9, pacemaker@oss.clusterlabs.org)
 CRM_DTD_VERSION="1.0"
 
+m4_pattern_allow([PKG_FEATURES])
 PKG_FEATURES=""
 HB_PKG=heartbeat
 
@@ -97,7 +98,7 @@ dnl ====================================
 cc_supports_flag() {
          local CFLAGS="$@"
          AC_MSG_CHECKING(whether $CC supports "$@")
-         AC_COMPILE_IFELSE([int main(){return 0;}] ,[RC=0; AC_MSG_RESULT(yes)],[RC=1; AC_MSG_RESULT(no)])
+         AC_COMPILE_IFELSE([AC_LANG_SOURCE([int main(){return 0;}])] ,[RC=0; AC_MSG_RESULT(yes)],[RC=1; AC_MSG_RESULT(no)])
          return $RC
 }
 
@@ -502,7 +503,6 @@ AC_PATH_PROGS(TAR, tar)
 AC_PATH_PROGS(MD5, md5)
 AC_PATH_PROGS(TEST, test)
 AC_PATH_PROGS(PKGCONFIG, pkg-config)
-AC_PATH_PROGS(XML2CONFIG, xml2-config)
 AC_PATH_PROGS(VALGRIND_BIN, valgrind, /usr/bin/valgrind)
 AC_DEFINE_UNQUOTED(VALGRIND_BIN, "$VALGRIND_BIN", Valgrind command)
 
@@ -671,17 +671,13 @@ dnl AC_CHECK_HEADERS(netinet/icmp6.h)
 dnl AC_CHECK_HEADERS(netinet/ip6.h)
 dnl AC_CHECK_HEADERS(netinet/ip_icmp.h)
 
-AC_MSG_CHECKING(for special libxml2 includes)
-if test "x$XML2CONFIG" = "x"; then
-   AC_MSG_ERROR(libxml2 config not found)
-else
-   XML2HEAD="`$XML2CONFIG --cflags`"
-   AC_MSG_RESULT($XML2HEAD)
-   AC_CHECK_LIB(xml2, xmlReadMemory)
-   AC_CHECK_LIB(xslt, xsltApplyStylesheet)
-fi
+PKG_CHECK_MODULES([XML], [libxml-2.0])
+PKG_CHECK_MODULES([XSLT], [libxslt])
+
+AC_CHECK_LIB(xml2, xmlReadMemory)
+AC_CHECK_LIB(xslt, xsltApplyStylesheet)
 
-CPPFLAGS="$CPPFLAGS $XML2HEAD"
+CPPFLAGS="$CPPFLAGS $XML_CFLAGS"
 
 AC_CHECK_HEADERS(libxml/xpath.h)
 AC_CHECK_HEADERS(libxslt/xslt.h)
