diff -Naur a/pengine/Makefile.am b/pengine/Makefile.am
--- a/pengine/Makefile.am	2014-07-23 13:24:08.887745910 +0200
+++ b/pengine/Makefile.am	2014-07-23 13:28:22.704843745 +0200
@@ -42,12 +42,9 @@
 
 ## binary progs
 halib_PROGRAMS	= pengine
-sbin_PROGRAMS	= ptest
 
 if BUILD_HELP
-man8_MANS =	ptest.8
-%.8:	%
-	help2man --output $@ --no-info --section 8 --name "Part of the Pacemaker cluster resource manager" $(top_builddir)/pengine/$<
+## removed the ptest functionality
 endif
 
 ## SOURCES
@@ -64,11 +61,6 @@
 # libcib for get_object_root()
 #		$(top_builddir)/lib/hbclient/libhbclient.la
 
-ptest_SOURCES	= ptest.c 
-ptest_LDADD	= $(COMMONLIBS)						\
-		$(top_builddir)/lib/cib/libcib.la			\
-		$(top_builddir)/lib/transition/libtransitioner.la
-
 install-exec-local:
 	$(mkinstalldirs) $(DESTDIR)/$(PE_STATE_DIR)
 	-chown $(CRM_DAEMON_USER) $(DESTDIR)/$(PE_STATE_DIR)
