From 1a84908d57291a82e35eaa9369144a398e2265bf Mon Sep 17 00:00:00 2001
From: Bian Naimeng <biannm@cn.fujitsu.com>
Date: Fri, 3 Jul 2015 16:30:25 +0900
Subject: [PATCH 4/4] pacemaker: fix header defs lookup

Signed-off-by: Bian Naimeng <biannm@cn.fujitsu.com>
---
 configure.ac | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/configure.ac b/configure.ac
index cb47011..d37eccf 100644
--- a/configure.ac
+++ b/configure.ac
@@ -141,9 +141,8 @@ extract_header_define() {
 	  Cfile=$srcdir/extract_define.$2.${$}
 	  printf "#include <stdio.h>\n" > ${Cfile}.c
 	  printf "#include <%s>\n" $1 >> ${Cfile}.c
-	  printf "int main(int argc, char **argv) { printf(\"%%s\", %s); return 0; }\n" $2 >> ${Cfile}.c
-	  $CC $CFLAGS ${Cfile}.c -o ${Cfile}
-	  value=`${Cfile}`
+	  printf "\"%s\":%s\n" $2 $2 >> ${Cfile}.c
+	  value=`$CC $CFLAGS -E ${Cfile}.c | grep \"$2\" | cut -f 2 -d ':' | sed 's,^",,' | sed 's,"$,,'`
 	  AC_MSG_RESULT($value)
 	  printf $value
 	  rm -rf ${Cfile}.c ${Cfile} ${Cfile}.dSYM ${Cfile}.gcno
-- 
1.8.4.2

